src/main/resources/db/changesets/20181203_studytable.yaml

```diff
@@ -0,0 +1,109 @@
+databaseChangeLog:
+  - changeSet:
+      id: study_init
+      author: jhert
+      changes:
+        - createTable:
+            tableName: study
+            columns:
+              - column:
+                  name: id
+                  type: UUID
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(1000)
```

Comment:
We should think about display names versus BQ (or other DB) object names.
A nice human-readable name that can accept international characters seems useful. That won't work in general as a name in a database system. BQ uses 1024 letter UTF-8 object names which is nice, but Redshift uses 127 byte restricted ASCII characters :(  So I think we might want to maintain a display name separate from the object name.